### DISCUSSION TITLE
Please Help: ComfyUI Mac issue ModuleNotFoundError: No module named 'yaml'

### DISCUSSION THREAD

**Discussion by deepvaid** [Comfy-Org/ComfyUI#929] [Q&A]
Hi I'm newbie here.

Im trying to install ComfyUI on M1pro book. but whenever I run python main.py I get response "ModuleNotFoundError: No module named 'yaml'

Though I have already rename "extra_model_paths.yaml.examples" to extra_model_paths.yaml. Can anyone please help what I'm doing wrong?

<img width="686" alt="Screenshot 2023-07-18 at 11 48 27 PM" src="https://github.com/comfyanonymous/ComfyUI/assets/36878165/9640248e-7442-4409-938f-e3d90cbb1b19">
<img width="633" alt="Screenshot 2023-07-18 at 11 48 51 PM" src="https://github.com/comfyanonymous/ComfyUI/assets/36878165/c13e8e73-0e52-409b-a3a2-ebc7708734a9">

**Comment by pinpin0121** [comment]
are ur confyui right?metooo

**Comment by ltdrdata** [comment]
Did you follow these installation instructions? 
https://github.com/comfyanonymous/ComfyUI?tab=readme-ov-file#apple-mac-silicon
